### PR TITLE
import hindi

### PR DESCRIPTION
**Pull request by GurpinderSandhu**
## Description

Adding hindi as a lang pack

### CODE REVIEW

File: localize-behavior.js [hunk 1]
```diff
@@ -69,6 +70,7 @@ D2L.PolymerBehaviors.Sequences.LocalizeBehaviorImpl = {
 					'fr': fr,
 					'fr-fr': frfr,
 					'fr-on': fron,
+					'hi':hi,
```
Comment (ianlaidlaw) [inline localize-behavior.js:73]: super mega tiny nit: space after the `:`

**Review by ianlaidlaw — APPROVED** [review]

File: localize-behavior.js (hunk 1 above)
Comment (GurpinderSandhu) [reply localize-behavior.js]: i mega agree

**Comment by brissons** [comment]
do we need to add something here as well (I am unfamiliar with serge)?
https://github.com/BrightspaceHypermediaComponents/sequences/blob/a91e77d89e6ebc3b2adbe6b85ebb25805436c7d0/d2l-sequences.serge.json

**Comment by ianlaidlaw** [comment]
> do we need to add something here as well (I am unfamiliar with serge)? https://github.com/BrightspaceHypermediaComponents/sequences/blob/a91e77d89e6ebc3b2adbe6b85ebb25805436c7d0/d2l-sequences.serge.json

I _think_ that is just for those languages that have multiple versions... like handling fallbacks? I think if this is the base `hi` language and there's no other, there's nothing we need to do now. I think?

**Comment by GurpinderSandhu** [comment]
> > do we need to add something here as well (I am unfamiliar with serge)? https://github.com/BrightspaceHypermediaComponents/sequences/blob/a91e77d89e6ebc3b2adbe6b85ebb25805436c7d0/d2l-sequences.serge.json
> 
> I _think_ that is just for those languages that have multiple versions... like handling fallbacks? I think if this is the base `hi` language and there's no other, there's nothing we need to do now. I think?

I would think so too, as serge was able to recognize this hi.js file and add a translation update PR to it already?

**Review by brissons — APPROVED** [review]

**Comment by brissons** [comment]
Thanks!

**Pull request merged**